Azure/azure-cli
# [Synapse] Add integration runtime related cmdlets

**Description**  

Add some new commands to manage integration runtime in synapse workspace.

- `az synapse integration-runtime` 

  - create/delete/show/list/update/upgrade
  - list-auth-key/regenerate-auth-key
  - get-monitoring-data/get-connection-info/get-status
  - sync-credentials

- `az synapse integration-runtime-node`

  - show/update/delete
  - get-ip-address

**Testing Guide**  

az synapse integration-runtime -h
az synapse integration-runtime-node -h

---

This checklist is used to make sure that common guidelines for a pull request are followed.
